tools/README.md

```diff
@@ -67,3 +67,23 @@ options:
 
 Example usage: change into your project directory and run `<path to BCR repo>/tools/migrate_to_bzlmod.py --target //foo:bar`
 ```
+
+## module_selector.py
+
+This script provides a way to select specific versions of Bazel modules from the Bazel Central Registry (BCR). It supports wildcard patterns for flexible module and version matching, as well as the option to randomly sample a percentage of the matching modules.
+
+```
+usage: module_selector.py [-h] [--registry REGISTRY] --select SELECT [--random-percentage PERCENTAGE]
```

Comment:
maybe the readme should have a `bazel run` command recommended instead? 

otherwise I get
```
tools/module_selector.py 
Traceback (most recent call last):
  File "/Users/alexeagle/Projects/bazel-central-registry/tools/module_selector.py", line 22, in <module>
    from registry import RegistryClient
  File "/Users/alexeagle/Projects/bazel-central-registry/tools/registry.py", line 33, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
```

Reply:
Done, added instructions on how to run with bazel